Tella/Data/Database/UwaziDatabase.swift

```diff
@@ -0,0 +1,199 @@
+//
+//  TellaUwaziTemplateDatabase.swift
+//  Tella
+//
+//  Created by Robert Shrestha on 9/14/23.
+//  Copyright © 2023 HORIZONTAL. All rights reserved.
+//
+
+import Foundation
+
+// MARK: - Methods related to UwaziTemplateProtocol
+extension TellaDataBase: UwaziTemplateProtocol {
+    func createTemplateTableForUwazi() {
+        let columns = [
+            cddl(D.cId, D.integer, primaryKey: true, autoIncrement: true),
+            cddl(D.cTemplateId, D.text),
+            cddl(D.cEntity, D.text),
+            cddl(D.cDownloaded, D.integer),
+            cddl(D.cUpdated, D.integer),
+            cddl(D.cFavorite, D.integer),
+            cddl(D.cServerName, D.text),
```

Comment:
Do you think that we need `cServerName` field in the database and we have `cServerId` as a foreign key ?